gnomad/utils/filtering.py

```diff
@@ -564,6 +565,7 @@ def filter_freq_by_meta(
     :param keep: Whether to keep or remove the items specified by `items_to_filter`.
     :param combine_operator: Whether to use "and" or "or" to combine the items
         specified by `items_to_filter`.
+    :param freq_meta_based_array_expr: Optional array based on freq meta expression to be filtered.
     :return: Tuple of the filtered frequency and frequency meta expressions.
```

Comment:
You might need to change return, I got warnings from my Pycharm.

Reply:
Good catch!